fe2-android/app/src/main/java/com/github/dedis/popstellar/ui/detail/LaoDetailViewModel.java

```diff
@@ -160,7 +161,7 @@ private boolean attendedOrOrganized(LaoView laoView, RollCall rollcall) {
     String firstLaoId = laoView.getId();
     try {
       PublicKey pk = wallet.generatePoPToken(firstLaoId, rollcall.getPersistentId()).getPublicKey();
-      return rollcall.getAttendees().contains(pk) || isOrganizer().getValue();
+      return rollcall.getAttendees().contains(pk) || Boolean.TRUE.equals(isOrganizer().getValue());
```

Comment:
It's a tad complicated I agree, the problem with the suggestion is that `isOrganizer()` returns a `LiveData` which wraps a `Boolean`. So this contruction allow us to handle the case where the Boolean is null (it would evaluate to `false`)